cvat/apps/redis_handler/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+
+class RedisMigration(models.Model):
```

Comment:
Have you considered storing information about Redis migrations in Redis? It seems to me that storing them in the database could lead to unwanted side effects. For example, if the DB is rolled back to an earlier snapshot, it could lead to some migrations being applied again.

Reply:
> Have you considered storing information about Redis migrations in Redis? 

I thought about that, but the same problem exists if Redis is reset. + I assumed that we could use these migrations to modify data stored in Redis on disk (since this service also supports Redis API), for instance, when we will need to update chunks/previews keys.

Reply:
> I thought about that, but the same problem exists if Redis is reset.

Does it? If Redis is reset to nothing, it should be safe to apply all migrations again - after all, that's what you would do when creating a fresh CVAT instance.

> I assumed that we could use these migrations to modify data stored in Redis on disk (since this service also supports Redis API), for instance, when we will need to update chunks/previews keys.

I'm not sure it's practical to implement such migrations. On-disk Redis is used as a cache, so if we need to change the schema in some way, we should be able to just ignore old cache entries and let them expire.

Reply:
done